darwin/DarwinMachine.h

```diff
@@ -21,6 +21,10 @@ typedef struct DarwinMachine_ {
    vm_statistics_data_t vm_stats;
    processor_cpu_load_info_t prev_load;
    processor_cpu_load_info_t curr_load;
+#ifdef CPUFREQ_SUPPORT
+   CpuFreqData cpu_freq;
+   bool cpu_freq_ok;
```

Comment:
By refactoring `CpuFreq_init` to return `CpuFreqData*` on success and `NULL` on failure, the `bool` can be omitted.